mars/services/scheduling/supervisor/autoscale.py

```diff
@@ -46,10 +46,10 @@ async def __post_create__(self):
             strategy_cls = getattr(importlib.import_module(module), name)
         else:
             strategy_cls = PendingTaskBacklogStrategy
-        from ..supervisor import GlobalSlotManagerActor
+        from ..supervisor import GlobalResourceManagerActor
 
         self.global_slot_ref = await mo.actor_ref(
-            GlobalSlotManagerActor.default_uid(), address=self.address
+            GlobalResourceManagerActor.default_uid(), address=self.address
```

Comment:
`global_slot_ref` maybe renamed to `global_resource_ref`.
And there are similar issues elsewhere, we should check them.